prisma/schema.prisma.example

```diff
@@ -0,0 +1,17 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+// The prisma models for the entire Postgres database schema are originally
+// generated by running `npx prisma db pull`. For the egghead database, this
+// required tweaking about a dozen errors in the generated models: type
+// mismatches (bigint -> int) and foreign key naming mismatches.
+//
+// Rather than sharing the schema publicly as part of this spike, I'll include
+// a private link to it separate from the PR. Just copy and paste that file
+// here to try it out.
```

Comment:
This is just a `.example` file, you'll want to create an actual `prisma/schema.prisma` file and paste the contents of the file I will share in Slack.

Reply:
I think we should build this UP versus pulling in our entire database schema.

Reply:
@joelhooks I agree—assuming we find it practical to talk directly to the original DB as well as a newer one with our modern schema. (Whether hosted Postgres, Fly, or Planetscale.)